helper/test/table.c

```diff
@@ -13,9 +13,9 @@ int main(int argc ODP_UNUSED, char *argv[] ODP_UNUSED)
 	odph_table_t tmp_tbl;
 	odph_table_ops_t *test_ops;
 	char tmp[32];
-	char ip_addr1[] = "12345678";
-	char ip_addr2[] = "11223344";
-	char ip_addr3[] = "55667788";
+	char key1[] = "1234";
```

Comment:
The commit message is now out-of-date since the code no longer contains any comment about key format. I think the real reason for this commit is to fix the incorrect key length passed to the table creation function.

Reply:
Currently the key length passed to f_create() is smaller than the array, but I wouldn't necessarily call that "incorrect".

This commit doesn't fix any functional problem, and is in that sense totally optional. The real reason for it is only to hopefully avoid some confusion.

I'll add the same in some form in the commit comment, which is otherwise correct, as far as I can tell, it's just missing the reason why the commit was made.

Reply:
The code was incorrect (or obfuscated, if you prefer) in the sense that if the 8-byte ipaddr strings did not differ from each other in the first 4 bytes, things did not work as intended. This commit fixes that.